checkov/terraform/checks/resource/aws/DocDBLogging.py

```diff
@@ -0,0 +1,21 @@
+from checkov.common.models.consts import ANY_VALUE
+from checkov.common.models.enums import CheckCategories
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceValueCheck
+
+
+class DocDBLogging(BaseResourceValueCheck):
+    def __init__(self):
+        name = "Ensure DocDB Logging is enabled"
+        id = "CKV_AWS_85"
+        supported_resources = ['aws_docdb_cluster']
+        categories = [CheckCategories.LOGGING]
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def get_inspected_key(self):
+        return "enabled_cloudwatch_logs_exports"
+
+    def get_expected_value(self):
+        return ANY_VALUE
```

Comment:
@DrFaust92 I believe security requirements are this should be `audit`. Please use `get_expected_values` to allow passing in case of `audit` or `profiler`, as per the [terraform docs](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/docdb_cluster).

Reply:
This is a list so it seems i'll need to fall back on a more explicit scan check. ill add a function to check a list of values later.

Reply:
@DrFaust92 We should be supporting a list of values already, `get_expected_value` for wanted vs `get_expected_values` for the possible list. Does that work for you?

Reply:
Nope, it can compare a single value to a list, the `enabled_cloudwatch_logs_exports` argument can be a list of values which i didnt find a builtin solution for.

Reply:
@DrFaust92 I understand. I don't want this to block your PR. Feel free to merge 👍